mne_sandbox/connectivity/mvar.py

```diff
@@ -0,0 +1,283 @@
+# Authors: Martin Billinger <martin.billinger@tugraz.at>
+#
+# License: BSD (3-clause)
+
+from __future__ import division
+import numpy as np
+import logging
+
+from scot.varbase import VARBase
+from scot.var import VAR
+from scot.connectivity import connectivity
+from scot.connectivity_statistics import surrogate_connectivity
+from scot.xvschema import make_nfold
+from scot.utils import acm
+
+from mne.parallel import parallel_func
+from mne.utils import logger, verbose
+
+
+# todo: I'm sure we can import generator from somewhere...
+def _generator():
```

Comment:
[Yes](http://stackoverflow.com/questions/6416538/how-to-check-if-an-object-is-a-generator-object-in-python) - `types.GeneratorType`.